audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/exception/ApplicationAsyncExceptionHandler.java

```diff
@@ -33,8 +34,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(ApplicationAsyncExceptionHandler.class);
 
     @Override
-    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object... objects) {
-        String parameters = Stream.of(objects)
+    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object[] objects) {
+        String parameters = Stream.of(Optional.ofNullable(objects).orElse(Stream.of().toArray()))
```

Comment:
```suggestion
        String parameters = Stream.of(Optional.ofNullable(objects).orElseGet(() -> new Object[0])
```
would make more sense, or:
```suggestion
        String parameters = Arrays.stream(Optional.ofNullable(objects).orElseGet(() -> new Object[0] {}))
```

Reply:
Updated.